amida-tech/indaba-client
# [INBA-972]  Color and Styling Overhaul

#### What does this PR do?

- adds `named_colors` and `semantic_colors` to indaba. this sets the color codes as specific values and then uses those variables throughout indaba. 

- Originally there was around 40 colors in indaba, now there is 20. 

- This although changes the core colors of indaba... so it looks different! 
we also pulled out some of the common components of the styling in indaba and created reusable utilities for them like `button`, `table`, `text-input`, etc.

- added more rules to style linting. it expects a specific order for value inputs, additionally it checks for hex/color codes that are not variables. 

#### Related JIRA tickets:
https://jira.amida.com/browse/INBA-972

#### How should this be manually tested?

Overall there needs to be testing of each page, since there was style changes everywhere. Make sure there are no errors in the build and that all the pages work. 

#### Background/Context

Feel free to look through the changes in the scss. Most of the changes are the ordering of the scss.

#### Screenshots (if appropriate):
